src/utils/redirects.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-env node */
 
-let redirects = require('./redirects-list.json')
+let redirects = require('../../redirects-list.json')
```

Comment:
I know Ivan suggested this but I'm not getting why this doesn't go in src/util/ along with the redirect module and tests.

Reply:
Never mind, I guess it makes sense to have it near server.js